PyInstaller/hooks/hook-scipy.py

```diff
@@ -0,0 +1,15 @@
+from PyInstaller.utils.hooks import get_module_file_attribute
+
```

Comment:
Blank line. Please also move the PyInstaller import to the end. BTW: platform will no be required when using compat.is_win.